frontend-student/src/context/authContext.jsx

```
@@ -11,7 +11,7 @@ const LoadingSpinner = () => {
   return (
     <div>
       Loading...
-      <div className="animate-spin h-8 w-8 border-4 border-t-4 border-blue-500 rounded-full"></div>
+      <div className='animate-spin h-8 w-8 border-4 border-t-4 border-blue-500 rounded-full'></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)